lib/core/modules/coderunner/codeRunner.js

```diff
@@ -37,12 +37,12 @@ class CodeRunner {
       RunCode.registerVar(varName, code);
     });
 
-    this.events.setCommandHandler('runcode:eval', (code, cb, dashboard = false) => {
+    this.events.setCommandHandler('runcode:eval', (code, cb, saveForConsole = false) => {
```

Comment:
I think forConsoleOnly is more clear. Up to you.

Reply:
updated ^^